code/modules/mob/living/carbon/human/species_types/squidpeople.dm

```diff
@@ -0,0 +1,40 @@
+/datum/species/squid
+    name = "Squidperson"
+    id = "squid"
+    default_color = "b8dfda"
+    species_traits = list(MUTCOLORS,EYECOLOR)
```

Comment:
add
`inherent_traits = list(TRAIT_RESISTHIGHPRESSURE)`
for high pressure resistance

Reply:
i know of that, but doesn't that just give stupid amounts of pressure resistance?
Like full immunity?